src/Umbraco.Core/Services/IMediaService.cs

```diff
@@ -369,5 +370,33 @@ public interface IMediaService : IService
         /// <param name="userId">Optional id of the user creating the media item</param>
         /// <returns><see cref="IMedia"/></returns>
         IMedia CreateMediaWithIdentity(string name, int parentId, string mediaTypeAlias, int userId = 0);
+
+        /// <summary>
+        /// Gets the content of a media as a stream.
+        /// </summary>
+        /// <param name="filepath">The filesystem path to the media.</param>
+        /// <returns>The content of the media.</returns>
+        Stream GetMediaFileStream(string filepath);

```

Comment:
Hrm... do any of these methods need to belong to this interface? `GetMediaFileStream`, `SetMediaFile`, `DeleteMediaFile`, `GenerateThumbnails`?